network/policy.go

```diff
@@ -83,3 +85,17 @@ func GetPortMappingPolicy(externalPort int, internalPort int, protocol string, h
 		Data: rawData,
 	}, nil
 }
+
+// GetLoopbackDSRPolicy creates a policy to support loopback direct server return.
+func GetLoopbackDSRPolicy(ip *net.IP) (Policy, error) {
+	value := fmt.Sprintf(`"Destinations" : ["%s"]`, ip.String())
```

Comment:
I believe hcn.OutboundNatPolicySetting structure should be used for constructing this policy rather than string manipulation, check out how GetPortMappingPolicy is constructing the policy.

Reply:
Much simpler that way, I updated this now thanks!